src/coreclr/jit/codegencommon.cpp

```diff
@@ -9412,3 +9429,7 @@ bool CodeGen::genCanOmitNormalizationForBswap16(GenTree* tree)
 
     return (cast->gtCastType == TYP_USHORT) || (cast->gtCastType == TYP_SHORT);
 }
+
+#undef RBM_ALLFLOAT_USE
+#undef RBM_FLT_CALLEE_TRASH_USE
+#undef CNT_CALLEE_TRASH_FLOAT_USE
```

Comment:
You don't need to worry about `#undef` at the end of a translation unit. I suppose it's ok, but unnecessary?